addons/project_timesheet_holidays/models/hr_holidays.py

```diff
@@ -75,10 +75,17 @@ def _validate_leave_request(self):
     def _timesheet_create_lines(self):
         self.ensure_one()
         vals_list = []
-        work_hours_data = self.employee_id.list_work_time_per_day(
-            self.date_from,
-            self.date_to,
-        )
+        if self.employee_id:
+            work_hours_data = self.employee_id.list_work_time_per_day(
+                self.date_from,
+                self.date_to,
+            )
+        else:
+            for employee_id in self.employee_ids:
+                work_hours_data = employee_id.list_work_time_per_day(
+                self.date_from,
+                self.date_to,
+            )
```

Comment:
Is that working as expecting?
Because work_hours_data will only contain the work time per day on the last employee

Reply:
Not working as expected at all 
![image](https://user-images.githubusercontent.com/88150098/135413812-f94a4425-cb16-42bd-85ff-3a05d7adb89a.png)
I updated to only check if there is an employee_id in the in the hr.leave as they are the hr.leave created when splitting a holiday with many employees at the time it is approved. Therefore the hr.leave with many employees has redundant information with the individual ones so no need to create lines in account.analytic.line
Thanks!

Reply:
Perfect! Thanks :-D